examples/text_classification/example_glue_dataset.py

```diff
@@ -0,0 +1,14 @@
+import autogluon as ag
+from autogluon import TextClassification as task
+
+dataset = task.Dataset(name='SST')
+predictor = task.fit(dataset, epochs=5, warmup_ratio=0.1, log_interval=400, seed=2, num_trials=100, dev_batch_size=8,
```

Comment:
why we need to specify so many arguments? can we just `task.fit(dataset)`

Reply:
definitely...